packages/contracts/contracts/plugin/PluginSetupProcessor.sol

```diff
@@ -373,17 +374,11 @@ contract PluginSetupProcessor is DaoAuthorizable {
             revert PluginAlreadyInstalled();
         }
 
-        validateSetupId(pluginInstallationId, setupId);
+        validatePreparedSetupId(pluginInstallationId, preparedSetupId);
 
-        bytes32 newSetupId = _getSetupId(
-            _params.pluginSetupRef,
-            ZERO_BYTES_HASH,
-            _params.helpersHash,
-            bytes(""),
-            PreparationType.None
-        );
+        bytes32 appliedSetupId = _getAppliedSetupId(_params.pluginSetupRef, _params.helpersHash);
 
-        pluginState.currentSetupId = newSetupId;
+        pluginState.currentSetupId = appliedSetupId;
```

Comment:
Is there any potential confusion with `currentSetupId` in regards to prepared/applied?

Reply:
I don't think so. appliedsetupid will always be currentSetupId.

Reply:
Then rename it to`currentAppliedSetupId`